### ISSUE TITLE
Save games/profiles from the Windows version are incompatible

### ISSUE THREAD

**Issue by vpltd-peterm** [virtual-programming/dirtshowdown-linux#1]
Addressing the concerns that have been voiced here:
https://www.reddit.com/r/linux_gaming/comments/3hu72e/bad_news_if_youre_trying_to_transfer_your_game/

Unfortunately the profiles from Windows versions of the game are incompatible. We were supplied with a slightly different build of the game from the Windows release - while they are network compatible and otherwise identical in content, the storage of profiles is slightly different.

**Comment by claell** [comment]
So Steam Cloud does not work either cross platform, right now?

**Comment by vpltd-peterm** [comment]
The saves are compatible between Linux and MacOS, but not compatible with the Windows version of the game. We specifically programmed Steam Cloud synchronisation to avoid any clash between the Windows saves and our Linux/MacOS saves.

**Comment by claell** [comment]
But I cannot sync safe gamed via Steam Cloud between Windows and Linux, right?

**Comment by vpltd-peterm** [comment]
No, they are not compatible

**Comment by claell** [comment]
So will savegames be compatible in the future (so that also cloud sync is possible)?

**Comment by vpltd-peterm** [comment]
It's unlikely. We were unable to get the exact build used for the production release of Dirt on Windows.

**Comment by claell** [comment]
That is sad. Why don't you get the build? Also there is the option of updating the Windows build to match the safefiles of Linux and Mac.

**Comment by vpltd-kgalaj** [comment]
Unfortunately there is no option of updating the Windows build, or getting the exact build used for Windows version.

**Comment by claell** [comment]
Then the only option would be to do reverse-engineering of the Windows safe-files to be able to use the same format for Linux and Mac safe-files, right?

**Comment by vpltd-kgalaj** [comment]
Correct. If making the saved games compatible between platforms was deemed cost-effective higher up, that would be the way to go.

**Comment by vpltd-kgalaj** [comment]
As a programmer, I also prefer to be able to give the best possible experience to gamers. Unfortunately, in business environment it's not always possible to achieve :(

**Comment by claell** [comment]
Yes, probably true.
